book/website/collaboration/stakeholder-engagement.md

```diff
@@ -0,0 +1,142 @@
+
+### Example resources
+- SuperBloom (formerly called Simply Secure) [Knowledge Base](https://simplysecure.org/knowledge-base/)
+- User Interface Design Basics at [usability.gov](https://www.usability.gov/what-and-why/user-interface-design.html)
+- Diataxis [framework for technical documentation](https://diataxis.fr)
+- Read the Docs [documentation hosting](https://readthedocs)
+- Nielsen Norman [heuristics](https://www.nngroup.com/articles/ten-usability-heuristics/)
+
+### RAM insights
+* UX and UI design are specialised skills that researchers are not expected to be familiar with, so consult available resources and bring in a professional team when possible!
+* Look for design firms that focus on coaching and prioritise user accessibility and an open, collaborative process. Working with designers on a fixed project will improve the UX of your tool in the immediate term, but learning these skills as a team will allow you to consider UX and apply design tools at each stage of your project's lifecycle.
+
+## Stakeholder Impact Assessments
+### Key question: How can we develop projects with continual sensitivity to the ways in which these impact stakeholders?
+
+Stakeholder Impact Assessments (SIAs) are a tool used to gain anticipatory insight into the project's likely impacts, defined as the possible harms and benefits.
+SIAs take the form of questionnaires addressing the ethical permissibility, transparency, accountability, and equity of projects.
+SIAs are used to identify and document the full range of potential impacts and provide project teams with a contextual awareness of the social environments impacted by their projects. 
+This activity can help teams steer the direction of their projects proactively to ensure that they support the wellbeing and sustainability of the individuals and communities they impact and mitigate any harms. 
+
+Evaluations of project impacts should ideally be done in collaboration with stakeholders. 
+In the context of SIAs, stakeholders can be both individuals and communities that are impacted or may impact projects.
+Engaging stakeholders in conducting SIAs helps secure the accuracy and integrity of SIA outcomes. 
+This is because gaps in identity and experience between project teams and impacted individuals may cause there to be differences between how team members view project impacts and how stakeholders experience them. 
+Facilitating proportionate stakeholder engagement and input throughout the AI lifecycle is a way to gain a richer understanding of the impacts that projects may have on stakeholders. 
+ 
+A SIA is not a one-off activity but an iterative one that occurs at key points throughout the design, development, and deployment stages of the [AI project lifecycle](https://alan-turing-institute.github.io/turing-commons/skills-tracks/rri/rri-101-index/), each time informing the direction of the project so that it continually adapts to changing contexts. 
```

Comment:
```suggestion
Conducting a SIA is not a one-off activity but an iterative one that occurs at key points throughout the design, development, and deployment stages of the [AI project lifecycle](https://alan-turing-institute.github.io/turing-commons/skills-tracks/rri/rri-101-index/), each time informing the direction of the project so that it continually adapts to changing contexts. 
```